INSTALL.rst

```diff
@@ -247,9 +247,9 @@ venv
   pip install -r requirements.txt
   # Install build dependencies of scikit-image
   pip install -r requirements/build.txt
-  # Build scikit-image from source
-  spin build
-  # The new version lives under `${PWD}/build-install/.../site-packages`.
+  # Install scikit-image in editable mode, which will make sure that
+  # scikit-image is recompiled if necessary on import
+  spin install -v
   # Test your installation
   spin test
   # Build docs
```

Comment:
Also, we can't build docs (a priori) if we haven't installed the docs dependencies first:
```suggestion
  # Build docs
  pip install -r requirements/docs.txt
```

Reply:
I think you actually can. `spin docs` should actually take care of that...

Reply:
Well, I couldn't when I tried the other day. I had to `pip install -r requirements/docs.txt` first. That's the only reason why I added this (off-topic) suggestion. :wink:

Reply:
Oh, you are right, good catch! `spin docs` actually isn't installing deps since https://github.com/scikit-image/scikit-image/pull/6940. That PR doesn't explain why we added the `--install-deps` option though and turned it off by default. And I also don't remember. :thinking:

Reply:
Adressed in 8f9aeaf59618cf3d34d1856656145ed1e4555985.

Reply:
Thanks, Lars!